scripts/script-tag.js

```diff
@@ -95,48 +95,51 @@
         backdrop.remove();
     }
 
-    // TODO: add preorder check
-    function isProductProrderedOrBackordered (virtualId, variantId) {
-        return new Promise(function(resolve, reject) {
-            jQueryBopis.ajax({
-                type: 'GET',
-                // need to update this endpoint to use correct endpoint for checking the product preorder availability
-                url: `${shopUrl}/admin/products/${virtualId}.json`,
-                crossDomain: true,
-                headers: {
-                    'Content-Type': 'application/json'
-                },
-                success: function (data) {
-                    if (data.product.tags.includes('Pre-Order') || data.product.tags.includes('Back-Order')) {
-                        resolve(data.product.variants.find((variant) => variant.id == variantId).inventory_policy === 'continue')
-                    }
-                    else {
-                        resolve(false)
-                    }
-                },
-                error: function (err) {
-                    reject(err)
-                }
-            })
-        })
+    async function getCurrentProduct() {
+        await jQueryBopis.getJSON(`${window.location.pathname}.js`, function(product) {
+            currentProduct = product;
+        });
+    }
+
+    async function isProductAvailable(variantSku) {
+        const hasInventoryOnShopify = jQueryBopis("input[class='hc_inventory']").val() > 0
```

Comment:
```suggestion
        const hasInventoryOnShopify = jQueryBopis("input.hc_inventory").val() > 0
```

Reply:
Check if it is not breaking when element is not found

Reply:
Sir, when the element is not found then the statement will return false and will not break.